### ISSUE TITLE
Question for the code of the DPT

### ISSUE THREAD

**Issue by SYSUykLin** [VITA-Group/FSGS#1]
Hello:
    Great paper. I saw that u utilize the pre-trained Dense Prediction Transformer (DPT) model for zero-shot monocular depth estimation, but I can not find that in the code. Could u specify that? 
   Thx

**Comment by zehaozhu** [comment]
Hi, the monocular depth of input images is estimated at https://github.com/VITA-Group/FSGS/blob/main/utils/camera_utils.py#L49. We also predict the monocular depth of pseudo views at https://github.com/VITA-Group/FSGS/blob/main/train.py#L124

**Comment by SYSUykLin** [comment]
Hello, I have a question for your code.
`depth_loss_pseudo = (1 - pearson_corrcoef(rendered_depth_pseudo, -midas_depth_pseudo)).mean()`
Why the midas_depth_pseudo is minus?
Thx

**Comment by zehaozhu** [comment]
Hi,

The depth estimated by midas is the inverse depth maps.(
https://pytorch.org/hub/intelisl_midas_v2/)
So we need a negative function to transform them back. So the logic applies
for https://github.com/VITA-Group/FSGS/blob/main/train.py#L109.

Best,
Zehao

**Comment by cuttini** [comment]
Hi,
why in the depth_loss calculation you add +200 on the midas_depth?
`        depth_loss = min(
                        (1 - pearson_corrcoef( - midas_depth, rendered_depth)),
                        (1 - pearson_corrcoef(1 / (midas_depth + 200.), rendered_depth))
        )`
[https://github.com/VITA-Group/FSGS/blob/36ac717ac5997b6a3dc11b47b31d85ba7e80232b/train.py#L107C24-L107C89](url)

Best,
Davide

**Comment by zehaozhu** [comment]
Hi,

We add 200 to make the computation more stable since sometimes midas_depth
may be less than zero.

Best,
Zehao

**Comment by Wang-Wenqing** [comment]
Hi, 
If I have a forward depth map, should I modify depth loss as:

        depth_loss = (1 - pearson_corrcoef(midas_depth, rendered_depth))
